app/brave_generated_resources.grd

```diff
@@ -460,6 +460,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         <message name="IDS_REPORT_BROKEN_SITE_MAC" desc="The Mac menu item to report a broken site in the Help menu.">
           Report a Broken Site
         </message>
+        <message name="IDS_MUTE_TAB_MAC" desc="The Mac menu item to toogle active tab's mute state">
```

Comment:
why this has to be MAC-specific? we already have a similar menu item in the context menu (I see it on Windows).

Reply:
Because this PR is for adding mute entry to macOS specific menu bar.
Other platform doesn't have menu bar.
<img width="311" alt="image" src="https://user-images.githubusercontent.com/6786187/221545019-675af9bb-530b-4c4b-ad68-647f6ef468bf.png">

Reply:
I understand, but this looks like one of these cases where we can reuse already existed translation :)

although MacOS version wants most words to be `Capitalized`, so not maybe we cannot do anything here..

cc @mkarolin for possible optimization of such cases.

Reply:
Ah, you concerned de-dup same strings.
I found that there is `IDS_ACCNAME_MUTE_TAB` that has `Mute tab`.
However, `MenuItemBuilder` gets string id ...